### PR TITLE
Added list of followers/following

### PR DESCRIPTION
**Pull request by adrean-hipe**
### OVERVIEW

- added follower list in the profile page
- added initial follower/following list page
- added follow/unfollow button in the list
- modified the routes to resource

### NOTES

- php artisan migrate

### SNIPPETS

- profile page with follower list button
image
- list in own profile
image
- other user's profile page with follower list button
image
- list in other profile
image

### CODE REVIEW

File: laravel-app/app/Http/Controllers/FollowController.php [hunk 1]
```diff
@@ -3,11 +3,30 @@
...
 class FollowController extends Controller
 {
-    public function follow(User $user): RedirectResponse
+    /**
+     * To display all the followers-following of the user
+     */
+    public function show(User $user): View
+    {
+        $followers = $user->followers()->get();
+        $followings = $user->following()->get();
```
Comment (hipe01mark) [inline laravel-app/app/Http/Controllers/FollowController.php:17]: Question
1.) Are you getting all of the followers of the specific user or getting all of the followers of the different users?
2.) Same question with the following?

$user->followers()->get(); ->>>> This means you are getting all of the followers of any users.
$user ->followers() ->>>>> This means you are getting all of the followers of the specific user.

File: laravel-app/resources/views/follow/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+					</div>
+				@endforeach
+			</div>
+		</div>
+	</div>
+</div>
+
+@include('partials._footer')
+
+@endsection
```
Comment (rafael-parayno) [inline laravel-app/resources/views/follow/show.blade.php:76]: Add new line

File: laravel-app/app/Http/Controllers/FollowController.php (hunk 1 above)
Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/FollowController.php:17]: $user->followers()->get();
sir doesn't this return all the followers of the instance I passed in the variable $user? So it correctly retrieves all the followers of this specific user. Is this correct sir?

According to: 
https://stackoverflow.com/questions/68017966/im-trying-to-get-the-followers-of-a-specific-user-with-eloquent-from-a-many-to

File: laravel-app/resources/views/follow/show.blade.php (hunk 2 above)
Comment (adrean-hipe) [reply laravel-app/resources/views/follow/show.blade.php]: done sir, added new line. Thank you ❤️

File: laravel-app/app/Http/Controllers/FollowController.php (hunk 1 above)
Comment (hipe01mark) [reply laravel-app/app/Http/Controllers/FollowController.php:17]: If it's properly working based on what you want, I will save this as resolved 🙇

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

File: laravel-app/app/Http/Controllers/FollowController.php [hunk 3]
```diff
@@ -3,11 +3,30 @@
...
 
 use App\Models\User;
+use Illuminate\Contracts\View\View;
 use Illuminate\Http\RedirectResponse;
 
 class FollowController extends Controller
 {
-    public function follow(User $user): RedirectResponse
+    /**
+     * To display all the followers-following of the user
```
Comment (bpocrafael) [inline laravel-app/app/Http/Controllers/FollowController.php:12]: please remove comments that it is already understandable, you over documenting your code

File: laravel-app/app/Http/Controllers/FollowController.php [hunk 4]
```diff
@@ -3,11 +3,30 @@
...
+
+        return view('follow.show', [
+            'user' => $user,
+            'followers' => $followers,
+            'followings' => $followings,
+        ]);
+    }
+
+    /**
+     * To follow other user account.
```
Comment (bpocrafael) [inline laravel-app/app/Http/Controllers/FollowController.php:27]: also this one

File: laravel-app/app/Http/Controllers/FollowController.php [hunk 5]
```diff
@@ -19,7 +38,10 @@ public function follow(User $user): RedirectResponse
         return redirect()->back()->with($success);
     }
 
-    public function unfollow(User $user): RedirectResponse
+    /**
+     * To unfollow a user account.
```
Comment (bpocrafael) [inline laravel-app/app/Http/Controllers/FollowController.php:42]: this one

Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/FollowController.php]: done sir, thank you ❤️

File: laravel-app/app/Http/Controllers/FollowController.php (hunk 4 above)
Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/FollowController.php]: done sir, thank you ❤️

File: laravel-app/app/Http/Controllers/FollowController.php (hunk 3 above)
Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/FollowController.php]: done sir, thank you ❤️

**Review by bpocrafael — APPROVED** [review]

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**